src/main/java/org/candlepin/dto/shim/ContentInfoTranslator.java

```diff
@@ -66,7 +67,8 @@ public ContentDTO populate(ModelTranslator translator, ContentData source, Conte
             throw new IllegalArgumentException("dest is null");
         }
 
-        dest.uuid(source.getUuid())
+        dest
+            .uuid(source.getId())
```

Comment:
Extraneous line break, plus we're setting the UUID to an incorrect value. We're better off not setting setting it to null rather than putting bad data in the field.

Reply:
Done.